migration-engine/connectors/mongodb-migration-connector/src/schema_calculator.rs

```diff
@@ -1,9 +1,10 @@
 use crate::schema::MongoSchema;
-use datamodel::Datamodel;
+use datamodel::{common::preview_features::PreviewFeature, Datamodel};
+use enumflags2::BitFlags;
 use mongodb::bson::{Bson, Document};
 
 /// Datamodel -> MongoSchema
-pub(crate) fn calculate(datamodel: &Datamodel) -> MongoSchema {
+pub(crate) fn calculate(datamodel: &Datamodel, preview_features: BitFlags<PreviewFeature>) -> MongoSchema {
```

Comment:
In the calculator, the preview feature will already have been feature gated, so we will only ever get `DESC` in the datamodel if it's enabled. I think we don't need that new parameter.